library/constant_time.c

```diff
@@ -47,9 +47,15 @@
 
 #include <string.h>
 #if defined(MBEDTLS_USE_PSA_CRYPTO)
-#define PSA_TO_MBEDTLS_ERR(status) PSA_TO_MBEDTLS_ERR_LIST(status,    \
-                                                           psa_to_ssl_errors,              \
-                                                           psa_generic_status_to_mbedtls)
+/* Define a local translating function to save code size by not using too many
+ * arguments in each translating place. */
+static int local_err_translation(psa_status_t status)
+{
+    return psa_status_to_mbedtls(status, psa_to_ssl_errors,
+                                 sizeof(psa_to_ssl_errors),
```

Comment:
Shouldn't this be?

```suggestion
                                 sizeof(psa_to_ssl_errors) / sizeof(psa_to_ssl_errors[0]),
```

(and the others)